ruby/google-protobuf.gemspec

```diff
@@ -32,6 +32,8 @@ Gem::Specification.new do |s|
     s.add_development_dependency "rake-compiler-dock", "= 1.2.1"
```

Comment:
rake-compiler and rake-compiler-dock may need an update to support 3.4.0 as well:

https://github.com/rake-compiler/rake-compiler-dock/releases/tag/v1.7.0

Reply:
This 1.2.1 version doesn't even support 3.3 so perhaps it's already outdated or not particularly relied upon for official builds, but probably good to update nonetheless.